tst.c

```diff
@@ -73,98 +51,79 @@ static void *tst_stack_pop(tst_stack *s)
  *  NULL on success (deleted), otherwise returns the address of victim
  *  if refcnt non-zero.
  */
-static void *tst_del_word(tst_node **root,
-                          tst_node *node,
-                          tst_stack *stk,
-                          const int freeword)
+void *tst_del_word(tst_stack *stk, const int freeword)
```

Comment:
Why did you change its visibility (`static`)?

Reply:
That is a mistake sorry. The issue fixed.